### DISCUSSION TITLE
Consider putting website / docs in this repo

### DISCUSSION THREAD

**Discussion by LegNeato** [bevyengine/bevy#2573] [Ideas]
I've seen most new committers (including myself) forgetting to include docs because they are in another repo.

Bevy is already a monorepo, please consider including the website or the book in it as well.

Additional benefits:

* Atomic commits for changes and updated docs
* Can use `skeptic` to test examples in the docs on CI
* Can easily include raw changelog on website

While all these can be done with multiple repos, they are easier when using a monorepo.

**Comment by IceSentry** [comment]
I'm not sure I understand what you mean by "docs are in another repo" all the documentation available at docs.rs is inlined directly in the code that you can generate using `cargo doc`. As for the website, it isn't a rust crate so it makes sense to not be in the same repo. Also, the CI pipeline already cheks all the docs and the examples as far as I know.

**Reply by LegNeato** [reply]
To clarify I mean the book / website. Many projects keep their book and website in the same repo--I am suggesting Bevy does the same

**Reply by IceSentry** [reply]
I think the issue is that the website tracks the latest release and not the main branch. So making any changes to the book would need to be done on another branch which would not make atomic commits possible. The book still has a lot of test done in CI to make sure the examples run and are compatible with the latest release of bevy.
